README.md

```diff
@@ -18,25 +18,23 @@ NOTE: Database is SQLite3 via SQLAlchemy
 
 1. Clone the repo (`git clone https://github.com/${your-repo-name}/dwellinglybackend.git`)
 2. Install Python ( https://realpython.com/installing-python/ )
-3. Install pipenv: `pip3 install --user pipenv`.
-   - Python version 3.6 or higher is required
-4. Seed the database
-   - Run: `python seed_db.py`
+3. Install [pipenv](https://pipenv.pypa.io/en/latest/#install-pipenv-today)
+4. Run `pipenv install -d`
+   - If you get the error `ImportError: cannot import name 'Feature' from 'setuptools'`, your setuptools version might be at 46 or later. You may be able to get it to work using version 45 (e.g. `pip3 install setuptools==45`)
```

Comment:
This shouldn't be an issue anymore once PR #67 is merged as zope.interface uses deprecated setuptools which causes this error.

Reply:
Let me know if you think we can remove line 23 or would you rather leave it in?